File: client/src/app/Classes/borrower.ts
export class Borrower {
    id: string;
    tz: string;
    ageCategory: number;
    firstName: string;
    lastName: string;
    phoneNumber: string;
    mail: string;

    constructor(id: string, tz: string, ageCategory: number, firstName: string
        , lastName: string, phoneNumber: string, mail: string) {
            this.id=id;
            this.tz=tz;
            this.ageCategory=ageCategory;
            this.firstName=firstName;
            this.lastName=lastName;
            this.phoneNumber=phoneNumber;
            this.mail=mail;
    }
}

File: server/project/DAL/Borrower.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL
{
    public class Borrower
    {
        public int Id { get; set; }
        public string Tz { get; set; }
        public int? CategoryId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Mail { get; set; }
        public virtual Category Category { get; set; }
        public virtual List<Lend> Lends { get; set; }


    }
}
